clients/client-sdk/src/clients/channel.ts

```diff
@@ -185,7 +180,7 @@ export class ChannelClient extends BaseClient {
       ...ascendingParam,
       limit,
       index,
-      hiddenParam
+      hidden
```

Comment:
please use hiddenParam since it must also be possible to have hidden as undefined

Reply:
ok isn't necessary
```
interface Te {
    hidden:boolean;
    num : number
}
const t: Te= {
    num:11
} as Te

const {hidden, num} = t
console.log(hidden)
```